sunpy/map/maputils.py

```diff
@@ -80,7 +80,20 @@ def map_edges(smap):
     return top, bottom, left_hand_side, right_hand_side
 
 
-@u.quantity_input
```

Comment:
is there a reason this was removed?

Reply:
I thought it was for unit inputs only, this function takes a Skycoord.

Reply:
sorry yeah I think you're right, ignore my comment